src/rgw/rgw_crypt.cc

```diff
@@ -453,22 +453,40 @@ class AES_256_CBC : public BlockCrypt {
         failed_to_get_crypto = true;
     }
     bool result = true;
-    unsigned char iv[AES_256_IVSIZE];
-    for (size_t offset = 0; result && (offset < size); offset += CHUNK_SIZE) {
-      size_t process_size = offset + CHUNK_SIZE <= size ? CHUNK_SIZE : size - offset;
-      prepare_iv(iv, stream_offset + offset);
-      if (crypto_accel != nullptr) {
+    static bool using_batch_mode = cct->_conf.get_val<bool>("plugin_crypto_accelerator_batch_mode");
```

Comment:
In fact,  whether support batch mode is determined crypto_accel . So we should add new api of crypto_accel  which whether support batch mode rather than use option to control.

Reply:
done. When `accelerator == "crypto_qat" && crypto_accel != nullptr`, we call the batch mode directly.